### PR TITLE
Update data-gov-uk-deployment.html.md to deploy to kubernetes cluster

### PR DESCRIPTION
**Pull request by kentsanggds**
https://trello.com/c/2qXYzZYb/1239-switchover-paas-apps-to-k8s-cluster

### CODE REVIEW

File: source/manual/data-gov-uk-operations.html.md [hunk 1]
```diff
@@ -69,56 +37,34 @@ The sync is normally done automatically using Sidekiq Scheduler. There may be ti
...
-cf restage find-data-beta-staging
-```
-
-Verify the new password is being used:
-
-```
-cf ssh find-data-beta-staging
-echo $VCAP_SERVICES
-```
+When you make changes to secrets they will not be automatically picked up by the apps so you have to run this command to pick up the new secrets `kubectl rollout restart deploy -n datagovuk`
```
Comment (sengi) [inline source/manual/data-gov-uk-operations.html.md:70]: Think we might have [reloader](https://github.com/stakater/Reloader) everywhere now.

File: source/manual/data-gov-uk-operations.html.md [hunk 2]
```diff
@@ -69,56 +37,34 @@ The sync is normally done automatically using Sidekiq Scheduler. There may be ti
...
-
-Log into the Zendesk using the username and password obtained from the previous step.  Reset the password through the UI.
-
-Update the password in secrets using [uups](http://cli.cloudfoundry.org/en-US/cf/update-user-provided-service.html):
+Get the existing credentials (username and password) from AWS secrets manager under `govuk/dgu/datagovuk`:
 
-```
-cf uups find-staging-secrets -p '{"key1": "value1", "key2": "value2"}'
-```
+Log into the Zendesk using the username and password obtained from the previous step.  Reset the password through the UI and copy the password to the clipboard.
```
Comment (sengi) [inline source/manual/data-gov-uk-operations.html.md:66]:
```suggestion
Log into Zendesk using the username and password obtained from the previous step.  Reset the password through the UI and copy the password to the clipboard.
```

File: source/manual/data-gov-uk-operations.html.md [hunk 3]
```diff
@@ -12,55 +12,23 @@ parent: "/manual.html"
...
 ## Accessing data.gov.uk
 
-### PaaS
+### Datagovuk kubernetes cluster
 
-1. Download and install [Cloud Foundry CLI](https://github.com/cloudfoundry/cli#downloads)
+Publish and Find are hosted on the same AWS cluster as GOV.UK but in its own namespace - `datagovuk`
 
-2. Sign into the PaaS. You can either do this via username and password, or single sign-on. After authenticating, a list of your accessible spaces will be shown, select `data-gov-uk`.
+So in order to get information about the apps you will need to add `-n datagovuk` to any `kubeclt` command or run this command - `$ kubectl config set-context --current --namespace=datagovuk`
```
Comment (sengi) [inline source/manual/data-gov-uk-operations.html.md:19]:
```suggestion
So to manage data.gov.uk apps you will need to add `-n datagovuk` to any `kubectl` command or run `kubectl config set-context --current --namespace=datagovuk` to make it the default.
```

**Review by sengi — APPROVED** [review]

File: source/manual/data-gov-uk-operations.html.md (hunk 1 above)
Comment (kentsanggds) [reply source/manual/data-gov-uk-operations.html.md:70]: does that mean that this no longer needs to be run manually? I'll take the line out then

File: source/manual/data-gov-uk-operations.html.md (hunk 3 above)
Comment (kentsanggds) [reply source/manual/data-gov-uk-operations.html.md:19]: ok

**Pull request merged**